src/eth/rpc/rpc_parser.rs

```diff
@@ -54,8 +60,28 @@ pub fn rpc_parsing_error<S: serde::Serialize>(message: S) -> ErrorObjectOwned {
     ErrorObjectOwned::owned(PARSE_ERROR_CODE, PARSE_ERROR_MSG, Some(message))
 }
 
-impl From<EthError> for ErrorObjectOwned {
-    fn from(error: EthError) -> Self {
-        rpc_internal_error(error.to_string())
+pub struct RPCError(anyhow::Error);
```

Comment:
`RpcError`

Reply:
Ah I see, I was not aware of the acronym naming convention. The revm library even has structures that don't follow this convention such as `revm::EVMData`. I'll fix this tommorow.

Reply:
Take jsonrpsee as reference:
https://docs.rs/jsonrpsee/0.21.0/jsonrpsee/?search=rpc